bin/proving-service/src/commands/update_workers.rs

```diff
@@ -2,15 +2,15 @@ use clap::Parser;
 use reqwest::Client;
 use serde::{Deserialize, Serialize};
 
-use crate::commands::ProxyConfig;
-
 // ADD WORKERS
 // ================================================================================================
 
 /// Add workers to the proxy
 #[derive(Debug, Parser, Clone, Serialize, Deserialize)]
 pub struct AddWorkers {
     workers: Vec<String>,
+    proxy_host: String,
+    proxy_update_workers_port: u64,
 }
```

Comment:
These are new command-line parameters, right? If so, we should mention them in the readme (under the "Updating workers on a running proxy" section).